auth-web/src/views/auth/RequestNameView.vue

```diff
@@ -56,6 +61,9 @@ import NumberedCompanyTooltip from '@/components/auth/common/NumberedCompanyTool
   }
 })
 export default class RequestNameView extends Vue {
+  private nroUrl = ConfigHelper.getNroUrl()
+  private learnMoreUrl = 'https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/' +
```

Comment:
is this string concatenation needed here?, ok to be a single string right?

Reply:
Definitely, will update this! In the Filings and Create UI's we have a max length on a line for the linters, so i've gotten used to that :)

Reply:
Fixed in next commit!